sources/cqlrt_common.c

```diff
@@ -2312,7 +2305,7 @@ cql_fetch_info cql_no_rows_row_info = {
 };
 
 // The most trivial empty result set that still looks like a result set
-cql_result_set_ref _Nonnull cql_no_rows_result_set() {
+cql_result_set_ref _Nonnull cql_no_rows_result_set(void) {
```

Comment:
This is peculiar... it should only be necessary on the function prototype.  I wonder if that's a new clang warning level or something.   Or maybe it's an xcode thing?   Anyway I have no objection to this.   If you tell me what the warning is I can turn it on.